packages/itwin/tree-widget/src/tree-widget-react/components/trees/models-tree/internal/FilteredTree.ts

```diff
@@ -37,9 +39,15 @@ interface ElementFilteredTreeNode extends BaseFilteredTreeNode {
 
 type FilteredTreeNode = GenericFilteredTreeNode | CategoryFilteredTreeNode | ElementFilteredTreeNode;
 
+interface GetElementsFromUnfilteredChildrenTreeProps {
+  childrenTree: ChildrenTree<any>;
```

Comment:
`unknown` should be preferred over `any` if we don't care about the type

```suggestion
  childrenTree: ChildrenTree<unknown>;
```